jcache/embedded/src/main/java/org/infinispan/jcache/annotation/InjectedCacheResolver.java

```diff
@@ -97,13 +120,34 @@ private JCacheManager toJCacheManager(EmbeddedCacheManager cacheManager) {
       return getCacheFromDefaultCacheManager(cacheName);
    }
 
-   private <K, V> Cache<K, V> getCacheFromDefaultCacheManager(String cacheName) {
-      Cache<K, V> cache = defaultJCacheManager.getCache(cacheName);
-      if (cache != null)
-         return cache;
+   private <K, V> Cache<K, V> getCacheFromDefaultCacheManager(final String cacheName) {
+      final Configuration defaultInjectedConfiguration = getBeanReference(Configuration.class);
```

Comment:
Hi @galderz , 
Infinispan produces a default configuration, which can be overriden[1].
 
[1] [19.2.2. Override the default embedded cache manager and configuration](http://infinispan.org/docs/7.0.x/user_guide/user_guide.html#_override_the_default_embedded_cache_manager_and_configuration)